vendor/bat-native-ads/src/bat/ads/internal/tokens/refill_unblinded_tokens/request_signed_tokens_url_request_builder.cc

```diff
@@ -63,11 +67,14 @@ std::vector<std::string> RequestSignedTokensUrlRequestBuilder::BuildHeaders(
 
   const std::string accept_header = "accept: application/json";
 
-  return {
-    digest_header,
-    signature_header,
-    accept_header
-  };
+#if defined(RPILL_ENABLED)
+  const std::string cache_header =
+      rpill::exec() ? "pragma: no-cache " : "pragma: no-cache";
```

Comment:
I still don't understand what's going on here. It seems like the cache header is the same regardless of the result or rpill enabled/disabled